packages/scandipwa/src/component/MyAccountOverlay/MyAccountOverlay.container.js

```diff
@@ -166,7 +177,19 @@ export class MyAccountOverlayContainer extends PureComponent {
             }
         }
 
-        if (newMyAccountState !== STATE_LOGGED_IN && pathname.includes(MY_ACCOUNT_URL)) {
+        /*
+        * Prevent unauthorized page access for
+        * signed-out users
+         */
+        const isInWhiteList = this.whiteList.reduce((result, url) => {
+            if (pathname.includes(url)) {
+                return true;
+            }
+
+            return result;
+        }, false);
```

Comment:
`reduce` is semantically incorrect here. Use `find`.

Reply:
@yeegor The work is in process. Anyways, appreciate noting